build/import-managed.sh

```diff
@@ -0,0 +1,86 @@
+#! /bin/bash
+
+set -e
+
+: ${MANAGED_KUBE?:MANAGED_KUBE must be set.}
+: ${MANAGED_CLUSTER_NAME?:MANAGED_CLUSTER_NAME must be set.}
+
+echo "* Attaching managed cluster ${MANAGED_CLUSTER_NAME} to hub"
+
+oc create ns ${MANAGED_CLUSTER_NAME}
+
+cat <<EOF | oc create -f -
+apiVersion: cluster.open-cluster-management.io/v1
+kind: ManagedCluster
+metadata:
+  name: ${MANAGED_CLUSTER_NAME}
+  labels:
+    cloud: auto-detect
+    vendor: auto-detect
+spec:
+  hubAcceptsClient: true
+EOF
+
+cat <<EOF | oc create -f -
+apiVersion: v1
+kind: Secret
+metadata:
+  name: auto-import-secret
+  namespace: ${MANAGED_CLUSTER_NAME}
+stringData:
+  autoImportRetry: "5"
+  kubeconfig: |-
+$(cat ${MANAGED_KUBE} | sed 's/^/    /g')
+type: Opaque
+EOF
+
+for i in {1..6}; do
+  err_code=0
+  echo "Waiting for ManagedCluster ${MANAGED_CLUSTER_NAME} to be available (${i}/6) ..."
+  oc wait managedcluster ${MANAGED_CLUSTER_NAME} \
+    --for condition=ManagedClusterJoined=True \
+    --for condition=ManagedClusterConditionAvailable=True \
+    --timeout 30s && break || err_code=$?
+done
+
+oc get managedcluster ${MANAGED_CLUSTER_NAME}
+
+if [[ "${err_code}" != "0" ]]; then
+  echo "ManagedCluster ${MANAGED_CLUSTER_NAME} failed to become available."
+  exit ${err_code}
+fi
+
+cat <<EOF | oc create -f -
+apiVersion: agent.open-cluster-management.io/v1
+kind: KlusterletAddonConfig
+metadata:
+  name: ${MANAGED_CLUSTER_NAME}
+  namespace: ${MANAGED_CLUSTER_NAME}
+spec:
+  applicationManager:
+    enabled: true
+  certPolicyController:
+    enabled: true
+  iamPolicyController:
```

Comment:
Maybe we don't need this `iamPolicyController`

Reply:
Good point!

Reply:
Actually, looks like it's required in the KlusterletAddonConfig CRD, FYI:
```
The KlusterletAddonConfig "managed-1" is invalid: spec.iamPolicyController: Required value
```